Sources/Client/Model/Message/Message+Requests.swift

```diff
@@ -65,10 +65,70 @@ public extension Message {
         Client.shared.unflag(message: self, completion)
     }
     
+    // MARK: - Translations
+    
+    /// Get the message text translated in language
+    /// Please call `translate` if this returns `nil` to get the message translated.
+    /// - Parameters:
+    ///   - locale: Destination locale. Defaults to `.current`
+    /// - Returns: The translation if it exists, or nil. Please call `translate` if this returns `nil` to get the message translated.
+    func translated(to locale: Locale = .current) -> String? {
+        guard let language = Language(locale: locale) else {
+            return nil
```

Comment:
I think it would be helpful to log an error message here with the reason why it failed. This can be quite tricky to debug.